src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_TIME_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int WORK_DAY_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int WORKING_HOURS_INDEX = 2;
+    private static final int INCOME_PER_HOUR_INDEX = 3;
+    private static final String SPLIT_BY_SPACE = " ";
```

Comment:
```suggestion
    private static final String SPACE_SEPARATOR = " ";
```

don't use verbs in field naming